web/src/components/UI/shadowContainer.tsx

```diff
@@ -0,0 +1,12 @@
+import styled from '@emotion/styled';
+import { Box, Flex } from 'rebass';
+
+
+const boxShadow = `box-shadow: 1px 3px 4px 4px #cbcbcb7d`;
```

Comment:
If this is a common box shadow, than one way of nicely organizing reusable styled components is to place them into a file called `styleTokens.js` and just have them exportable and then import them into files such as this. I use style tokens for common styles like `border-radius`, `border`, `box-shadow` etc.